### ISSUE TITLE
Dashboard alignment css

### ISSUE THREAD

**Issue by sadortun** [orocrm/crm#170]
Hi!

Using French translations, the text in the widget button wraps:
![image](https://cloud.githubusercontent.com/assets/794449/15170441/d3d799e4-1713-11e6-8a39-bc7ffd5a09e3.png)

Quick fix (if you like hardcoding heights ):  
```css
.quick-launchpad-toolbar > li {
/* ..... */
    height: 120px;
    min-height: 120px;
    overflow: auto;
}
```

**Comment by Grygir** [comment]
Thank you for reporting the issue. We will fix it in coming release.

**Comment by artml** [comment]
@sadortun again, the fix is in the master already and will be shipped in 1.10.
